src/components/Banners/SpruceConfigBanner.tsx

```diff
@@ -0,0 +1,9 @@
+import { useSpruceConfig } from "hooks";
+import { SiteBanner } from "./SiteBanner";
+
+export const SpruceConfigBanner = () => {
```

Comment:
Imo this name isn't obvious that it's the sitewide admin banner... maybe it could be called `AdminBanner`, or remain as `SiteBanner` and rename the current `SiteBanner` to just `Banner`? 🤔